src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -21,9 +44,39 @@ const BobaModalform = ({ milk, setMilk }: { milk: string; setMilk: (milk: string
         <Col xs={12} md={8}>
           Milk(Required)
```

Comment:
Use something like Astrid `*`
https://stackoverflow.com/questions/23141854/adding-asterisk-to-required-fields-in-bootstrap-3